mllib/src/main/scala/org/apache/spark/ml/ann/Layer.scala

```diff
@@ -438,6 +478,27 @@ private[ml] object FeedForwardTopology {
     }
     FeedForwardTopology(layers)
   }
+
+  /**
+   * Creates a multi-layer perceptron for regression
+   *
+   * @param layerSizes sizes of layers including input and output size
```

Comment:
Yeah I wanted to check - is there a reason we can't support n-dimension output? It's less common but the MLP can support it.